pkg/controller/statefulset/stateful_pod_control_test.go

```diff
@@ -63,7 +63,7 @@ func TestStatefulPodControlCreatesPods(t *testing.T) {
 	}
 	for i := range events {
 		if !strings.Contains(events[i], v1.EventTypeNormal) {
-			t.Errorf("Expected normal events found %s", events[i])
+			t.Errorf("Expected normal event found %s", events[i])
```

Comment:
The original wording here is weird, particularly the missing punctuation before "found". While you're at it, would you mind updating the general style of these error messages? That would make it more worthwhile to go through the process of review and presubmit checks.

For example, I think this error would be more clear as: `"found unexpected non-normal event: %s"`